### PR TITLE
different button sizes has now icons

### PR DESCRIPTION
**Pull request by brianbodollo**
image

### CODE REVIEW

**Comment by srph** [comment]
I think you should move this to With Icons examples.

screen shot 2015-10-23 at 12 57 32 am

**Comment by brianbodollo** [comment]
image

**Comment by srph** [comment]
Just remove the heading "With Icons Inside in Different Sizes", and just put it below the examples.

**Comment by brianbodollo** [comment]
:+1:

File: docs/sections/Docs-Buttons/index.jsx [hunk 1]
```diff
@@ -74,21 +73,21 @@ export default class Buttons extends React.Component {
...
-              <i className="icon icon-user3" /> Button <i className="icon icon-user3" />
-            </button>
+            <button className="btn -emerald">Button <i className="icon icon-user3" /> </button>
+          </div>
 
-            <button className="btn -emerald">
-              Button <i className="icon icon-user3" />
-            </button>
+          <div className="doc-btn-group">
+            <button className="btn -xs -emeral+d"> <i className="icon icon-user3" /> Button </button>
```
Comment (srph) [inline docs/sections/Docs-Buttons/index.jsx]: Oops, typo right there.

Comment (brianbodollo) [inline docs/sections/Docs-Buttons/index.jsx]: done

**Comment by srph** [comment]
Looks good. We'd like to maintain a clean commit history, so next time.. thanks.

**Comment by srph** [comment]
Referencing https://github.com/dev-onenetworkecommerce/aia/issues/100

**Pull request merged**